guides/setup.md

```diff
@@ -0,0 +1,95 @@
+---
+title: DustJS | Setup
+layout: guides
+permalink: /guides/setup/
+---
+
+# Installing Dust
+
+## npm
+
+To install as a Node module:
+
+```
+npm install --save dustjs-linkedin
+```
+
+Dust should work with any version of Node or io.js newer than 0.8.12.
+
+The command-line compiler is located at `./node_modules/.bin/dustc`
```

Comment:
I think that's more of Node knowledge than Dust documentation, really.

You would know that something in .bin can be globalized with `-g`.